src/main/java/at/hannibal2/skyhanni/data/PetAPI.kt

```diff
@@ -13,19 +47,128 @@ object PetAPI {
         "Pets(?: \\(\\d+/\\d+\\) )?",
     )
 
+    var pet: PetData? = null
+        private set
+    var inPetMenu = false
```

Comment:
since this has no modified event priority at InventoryFullyOpenedEvent, make it private as its not useablef or other open events